PUBLISHING.md

```diff
@@ -5,71 +5,69 @@ the crates in this repository to [crates.io](https://crates.io/).
 
 **It is mostly intended for maintainers of the uefi-rs project.**
 
-## Bumping the crate versions
+## TL;DR
+- Each release should match the [`CHANGELOG.md`].
+  The changelog needs to be up-to-date before any release.
+- Each release should have a git tag like `uefi-raw-v0.4.0` or `uefi-v0.25.0`.
+- Each release should contain something useful. If we just bump `uefi` in
+  `uefi-services`, there is no real value-add (as long as the new version is
```

Comment:
I would say `no real added value` sounds like a more natural expression in this context.